Sklep/ReceiptPosition.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Sklep
+{
+    public partial class ReceiptPosition : Control
+    {
+        public event EventHandler<int> RemoveButtonClick;
+
+        private Label _productLongName;
+        private Label _pricePerUnit;
+        private Label _price;
+        private NumericUpDown _amount;
+        public Button removeButton;
+        public decimal PricePerUnit
+        {
+            get
+            {
+                return Decimal.Parse(_pricePerUnit.Text);
+            }
+            set
+            {
+                _pricePerUnit.Text = value.ToString();
+            }
+        }
+        public decimal Amount
+        {
+            get
+            {
+                return _amount.Value;
+            }
+            set
+            {
+                _amount.Value = value;
+            }
+        }
+        public string ProductLongName
+        {
+            get
+            {   
+                return _productLongName.Text; 
+            }
+            set
+            {
+                _productLongName.Text = value;
+            }
+        }
+        public int id;
+        public string barcode;
+      
+        
+        public ReceiptPosition()
+        {
+            InitializeComponent();
+            _productLongName = new Label()
+            {
+                Text = "Product",
+                Location = new System.Drawing.Point(0, 4),
```

Comment:
skoro mamy using System.Drawing to potrzebne jest rozwijanie typu do pełnego `System.Drawing.Point`? nie wystarczy `Point`? To samo do Size.